openstudiocore/src/osversion/VersionTranslator.cpp

```diff
@@ -3004,9 +3004,36 @@ std::string VersionTranslator::update_1_10_0_to_1_10_1(const IdfFile& idf_1_10_0
   IdfFile targetIdf(idd_1_10_1.iddFile());
   ss << targetIdf.versionObject().get();
 
+  auto zones = idf_1_10_0.getObjectsByType(idf_1_10_0.iddFile().getObject("OS:ThermalZone").get());
+
   for (const IdfObject& object : idf_1_10_0.objects()) {
     auto iddname = object.iddObject().name();
-    if (iddname == "OS:Sizing:Zone") {
+
+    if (iddname == "OS:ThermostatSetpoint:DualSetpoint") {
```

Comment:
@kbenne what about other thermostat types?  not worried about them because they aren't common?